base_location_nuts/models/res_partner_nuts.py

```diff
@@ -24,7 +25,10 @@ class ResPartnerNuts(models.Model):
     # Parent hierarchy
     parent_id = fields.Many2one(comodel_name='res.partner.nuts',
                                 ondelete='restrict')
-    children = fields.One2many(comodel_name='res.partner.nuts',
-                               inverse_name='parent_id')
-    parent_left = fields.Integer('Parent Left', select=True)
-    parent_right = fields.Integer('Parent Right', select=True)
+    child_ids = fields.One2many(
+        'res.partner.nuts',
+        'parent_id',
+        "Children",
+        oldname="children")
```

Comment:
There's no need of oldname in one2many fields, as they don't have a real column in the DB.